source/includes/_endpoint-dataset.md

```diff
@@ -486,6 +486,16 @@ in the API.
 
 Categorical variables will be exported with their name instead of their value.
 
+You may provide a filter of your own choosing by adding a zcl json string to the "filter" parameter
+The zcl is supplied in the same format as regular dataset filters.  Here is an example url with filter on a
+categorical variable: 
+
+`/datasets/{id}/export/csv/?filter={"function": "==", "args": [{"variable": "000000"}, {"value": 1}]}`
+
+You can provide a selection of variables with the "where" parameter.  Here is how you would limit the export to
+a single variable:
+
+`/datasets/{id}/export/spss/?where={"function": "identify", "args": [{"id": ["000000"]}]}`
 
```

Comment:
Great. Is there a functional way for me to do something like export all hidden variables, or all of my private variables, or all categorical variables, or some other metadata characteristic? Not requiring that those be implemented, but if they do exist already, please doc briefly.